File: Java/Pt1Exercise7/PronicNumbers.java
import java.util.Scanner;

class PronicNumbers{
	public static void main(String args[]){

	Scanner scanner =  new Scanner(System.in);
	System.out.print("Enter number of pronic numbers: ");
	int numbers = scanner.nextInt();

	for (int i = 1; i <= numbers ;i++){
		System.out.print(i*(i+1) + " ");
	}
	System.out.println();
	}
}

File: Python/Pt1Exercise7/pronicnumbers.py
"""
 Exercise 7 from the Python tutorial Part 1 on:
 https://codeandwork.github.io/courses/prep/pythonTutorial1.html
"""

numbers = int(input("Enter number of pronic numbers:"))
for n in range(1, numbers+1):
    print(n*(n+1), end=" ")
